source/styles.scss

```diff
@@ -24,6 +24,7 @@
 
 // USWDS components.
 @import '_patterns/03-uswds/**/*.scss';
+@import 'packages/form-controls';
```

Comment:
If we end up keeping Sass partials for the USWDS section, I think it should come after importing from USWDS directly so that they can easily override them without needing to get a higher specificity.

Reply:
Yes, I'm fine with that. I was assuming I'd need to update this one way or another as part of the merge once #18 was finalized and merged.